lib/rules/check-hbs-template-literals.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
       const extendedOptions = context.options[1]
       const filename = extendedOptions.ConfigFile
       if (filename && fs.existsSync(filename)) {
-        config = JSON.parse(fs.readFileSync(filename))
+        config = require(filename)
         linter = new TemplateLinter({config})
```

Comment:
I don't fully understand why we don't let the linter find it's own config 🤔

Reply:
It is coming from #7, I guess it is due to missing support of `overrides` section?